NobleJoseph580/Perumpanachy

```diff
@@ -0,0 +1,3 @@
+Perumpanachy, Chaganaserry 
+9.485496,76.591023
+(https://user-images.githubusercontent.com/116492843/197390683-3a98d7b9-4f1f-46c6-b90d-eaabcb949a93.jpg
```

Comment:
- The folder structure is wrong. The folder should be within the `data` folder.
- Remove the '(' at the 3rd line